### ISSUE TITLE
Slow rstudio build

### ISSUE THREAD

**Issue by 22459** [NixOS/nixpkgs#34606]
cc @ehmry 

It seems to me that even though enableParallelBuild is set to true for rstudio, At least for the rstudio-core component it still only seems to build on one cpu core?

I'm building it with this subexpression that I pass to buildInputs:
```
rstudio =
    let
        config = { packageOverrides = pkgs: { rstudio = pkgs.libsForQt5.callPackage <unstable/pkgs/applications/editors/rstudio> { boost = nixos.boost163; };};};
    in
        #cannibalizing the call from all-packages to mix versions
        (import <nixos> { inherit config; }).callPackage <unstable/pkgs/development/r-modules/wrapper-rstudio.nix> {
           packages = rpackages;
           recommendedPackages = with pkgs.rPackages; [ boot class cluster codetools foreign KernSmooth lattice MASS Matrix mgcv nlme nnet rpart spatial survival ];
           };
```

Unstable is head, nixos is 17.09 .

**Comment by 22459** [comment]
Separate issue, any idea why that version of rstudio reports itself as 0.99? I though something was wrong with the nix stuff, but I think the rstudio is in fact 1.1.414, but its reporting it's version wrong for some reason.

**Comment by sifmelcara** [comment]
Can't reproduce. Maybe your system is configured to only use one core? See https://nixos.org/nixos/options.html#nix.buildcores
Or you can pass `--cores 4` to `nix-build` or `nix-shell` to force them build the nix expression with 4 cores.

**Comment by 22459** [comment]
Most things take advantage of the 4 cores I have. I don't. I guess I'll just have to pay attention until I notice something.

**Comment by 22459** [comment]
Hm. You're probably right. I may just have been inconsistent in passing cores. Thanks.

**Comment by sifmelcara** [comment]
Yes I can reproduce the version number issue. Just created a PR to fix that.

**Comment by 22459** [comment]
I should probably open another issue for this... @sifmelcara , does the broom button for clearing plots work for you? It doesn't for me, Also if it try to close rstudio with unsaved changes selecting "no" (for "don't save") in the popup dialog does nothing - it just goes back to rstudio.

**Comment by sifmelcara** [comment]
Cannot reproduce both issue (I even tried to to reproduce them in a clean vm). Maybe it have something to do with your desktop environment or `~/.config/RStudio`?
